WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -82,6 +82,11 @@ - (instancetype)init
                                                  selector:@selector(helpshiftUnreadCountUpdated:)
                                                      name:HelpshiftUnreadCountUpdatedNotification
                                                    object:nil];
+
```

Comment:
When I was looking at WPTabBarController, I noticed we don't do anything to remove the observers that are added. I understand this is fine since we treat this class as a singleton. My paranoia is kicking in a bit and I'm wondering if at some future point we might change that.  Would it make sense to go ahead and add a `dealloc` method to `[[NSNotificationCenter defaultCenter] removeObserver:self]`?  Just a though, not a necessary change.